Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1217,6 +1229,18 @@ public extension Purchases {
             throw NewErrorUtils.purchasesError(withUntypedError: error).asPublicError
         }
     }
+
+    func redeemWebPurchase(
+        webPurchaseRedemption: WebPurchaseRedemption,
+        completion: @escaping (CustomerInfo?, PublicError?) -> Void
+    ) {
+        self.purchasesOrchestrator.redeemWebPurchase(webPurchaseRedemption: webPurchaseRedemption,
+                                                     completion: completion)
+    }
+
+    func redeemWebPurchase(_ webPurchaseRedemption: WebPurchaseRedemption) async -> WebPurchaseRedemptionResult {
+        return await self.purchasesOrchestrator.redeemWebPurchase(webPurchaseRedemption)
```

Comment:
should this be a throwing function instead, and have the errors be thrown instead of a valid result? That would feel more Swift-y to me, cc @MarkVillacampa for a swiftier brain